libtransmission/transmission.h

```diff
@@ -501,9 +502,9 @@ enum tr_direction
 ****  Primary session speed limits
 ***/
 
-void tr_sessionSetSpeedLimit_Bps(tr_session*, tr_direction, unsigned int bytes_per_second);
-void tr_sessionSetSpeedLimit_KBps(tr_session*, tr_direction, unsigned int kilo_per_second);
-unsigned int tr_sessionGetSpeedLimit_KBps(tr_session const*, tr_direction);
+void tr_sessionSetSpeedLimit_Bps(tr_session*, tr_direction, tr_speed_t bytes_per_second);
+void tr_sessionSetSpeedLimit_KBps(tr_session*, tr_direction, tr_speed_t kilo_per_second);
```

Comment:
Sample suggested new version:

```suggestion
void tr_sessionSetSpeedLimit_Bps(tr_session*, tr_direction, tr_bytes_per_second_t limit);
void tr_sessionSetSpeedLimit_KBps(tr_session*, tr_direction, tr_kilobytes_per_second_t limit);
```